src/component/feature/FeatureToggleList/FeatureToggleListActions/FeatureToggleListActions.tsx

```diff
@@ -67,6 +80,7 @@ const FeatureToggleListActions = ({
                         updateCurrentProject={project =>
                             setFilter(prev => ({ ...prev, project }))
                         }
+                        // FIXME: styles
                         style={{
                             textTransform: 'lowercase',
```

Comment:
@FredrikOseberg I don't understand this. Why `lowercase` the project name?

Reply:
Good question. I'm not sure.